### PR TITLE
Test conventions

### PR DESCRIPTION
**Pull request by licarm**
## Description
Updating tests that we've already completed

### CODE REVIEW

File: packages/riipen-ui/src/components/Avatar.test.jsx [hunk 1]
```diff
@@ -74,6 +75,15 @@ describe("<Avatar>", () => {
           .hasClass(classVariant[0])
       ).toEqual(true);
     });
+
+    it("appends higher order values to classes prop with withClass decorator", () => {
+      const wrapper = mount(<Avatar />);
```
Comment (imogenem) [inline packages/riipen-ui/src/components/Avatar.test.jsx:80]: Let's make these all the same
```
const classes = ["riipen", "riipen-avatar"];

const wrapper = mount(<Avatar />);

expect(wrapper.find("Avatar").props().classes).toEqual(classes.sort());
```

File: packages/riipen-ui/src/components/FormGroup.test.jsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe("<FormGroup>", () => {
       ).toEqual(true);
     });
 
-    it("appends higher order values to default classes prop with withClass decorator", () => {
+    it("appends higher order values to classes prop with withClass decorator", () => {
       const wrapper = mount(<FormGroup />);
```
Comment (imogenem) [inline packages/riipen-ui/src/components/FormGroup.test.jsx:51]: Let's make these all the same
```
const classes = ["riipen", "riipen-formgroup"];

const wrapper = mount(<FormGroup />);

expect(wrapper.find("FormGroup").props().classes).toEqual(classes.sort());

File: packages/riipen-ui/src/components/List.test.jsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ describe("<List>", () => {
       ).toEqual(true);
     });
 
-    it("appends higher order values to default classes prop with withClass decorator", () => {
+    it("appends higher order values to classes prop with withClass decorator", () => {
       const wrapper = mount(<List />);
```
Comment (imogenem) [inline packages/riipen-ui/src/components/List.test.jsx:51]: Let's make these all the same
```
const classes = ["riipen", "riipen-list"];

const wrapper = mount(<List />);

expect(wrapper.find("List").props().classes).toEqual(classes.sort());

**Review by jordanell — APPROVED** [review]
Awesome job.

**Pull request merged**